drivers/syslog/Kconfig

```diff
@@ -140,11 +140,35 @@ config SYSLOG_INTBUFSIZE
 
 comment "Formatting options"
 
+config SYSLOG_RFC5424
+	bool "RFC5424 syslog protocol"
+	default n
+	---help---
+		Causes syslog logs to follow RFC 5424 (syslog protocol).
```

Comment:
Please explain for a newbie what is means :-)
So, instead of putting only the RFC number, please use sometime like:
        bool "Use modern syslog message format (RFC 5424)"
 	---help---
		Enable to option to force syslog output to follow the RFC 5424 format,
		which includes structured data, timestamps, and standardized
		severity levels. This format is more detailed and suitable
		for advanced logging systems.

Reply:
Great point! I'll revise that!